### PR TITLE
Update index.rst

### PR DESCRIPTION
**Pull request by LappleApple**

### CODE REVIEW

**Comment by danpersa** [comment]
LGTM :+1:

**Comment by jmcs** [comment]
:+1:

**Comment by hjacobs** [comment]
:+1: yeah, GitHub PR discussions instead of Google Doc :smile:

File: index.rst [hunk 1]
```diff
@@ -2,28 +2,27 @@
 Zalando Open Source Principles
 ==============================
 
-.. note::
```
Comment (hjacobs) [inline index.rst]: so we are now FINAL? :+1:

File: index.rst [hunk 2]
```diff
@@ -2,28 +2,27 @@
...
 * **Stay Secure**: Make sure your project doesn’t include Zalando specifics, such as credentials and private identifiers.
-* **Ask for Help**: Find colleagues to brainstorm your project and review your work.
+* **Ask for Help**: Find colleagues to brainstorm ideas for your project and to review your work.
 * **Promote**: Tell the world about your projects via blog posts, social media and conference talks.
-* **Join the Open Source Guild**: You are invited to help us make open source stronger at Zalando!
+* **Join the Open Source Guild**: Help us make open source stronger at Zalando!
+
+.. note::
 
+    Like all open source initiatives, this document remains a work in progress.
```
Comment (hjacobs) [inline index.rst]: OK, haven't seen this note..

**Pull request merged**